### PR TITLE
8800: Updates policies, specs

### PR DESCRIPTION
**Pull request by meltheadorable**

### CODE REVIEW

File: spec/factories/users.rb [hunk 1]
```diff
@@ -48,20 +48,18 @@
...
 
     trait :member do
-      transient do
-        division { create(:division) }
-      end
+      profile { create(:person, :with_member_access, :with_password) }
 
       after(:create) do |user, evaluator|
-        user.add_role :member, evaluator.division if evaluator.division.present?
+        user.add_role :member, (evaluator&.division || create(:division))
```
Comment (arisah-ijeoma) [inline spec/factories/users.rb]: cool

File: spec/policies/documentation_policy_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    permit_all
+  end
+
+  context "with member user" do
+    let(:user) { create(:user, :member) }
+
+    permit_actions [:show]
+    forbid_actions [:create, :edit, :update, :new]
+  end
+end
```
Comment (arisah-ijeoma) [inline spec/policies/documentation_policy_spec.rb]: 👍

**Comment by arisah-ijeoma** [comment]
This work is nice. I love the refactors but note_policy is broken

**Comment by meltheadorable** [comment]
Yeah, it's not broken for me locally. There's something weird going on with the tests here. I might just undo all the changes I made to the suite and find a new way to test this.

**Pull request closed without merging**